envconfig/config.go

```diff
@@ -266,7 +269,27 @@ func LoadConfig() {
 		}
 	}
 
-	KeepAlive = clean("OLLAMA_KEEP_ALIVE")
+	ka := clean("OLLAMA_KEEP_ALIVE")
+	if ka != "" {
```

Comment:
small nit: I'd probably shove this into a helper function just to keep the size of `LoadConfig()` down.